app/views/records/edit_fields/_qualification_level.html.erb

```diff
@@ -3,5 +3,12 @@
 
 <% model = f.object.model.class.to_s.underscore %>
 
-<%= render 'hyku_addons/qualification/qualification', f: f, model: model, qualification_service: qualification_level_service,
-    field_name: 'qualification_level', css_class_name: 'qualification-level'%>
+<% options = {
+    f: f,
+    model: model,
+    qualification_service: qualification_level_service,
+    field_name: 'qualification_level',
+    css_class_name: 'qualification-level'
+} %>
+
+<%=render 'hyku_addons/qualification/qualification', options %>
```

Comment:
You need a space after the opening erb tag and you should be using double quotes: 

`<%= render "hyku_addons/qualification/qualification", options %>`